tidb-lightning.toml

```diff
@@ -79,6 +79,16 @@ addr = "127.0.0.1:8287"
 #  - ignore: keep the old record and ignore the new record (i.e. insert rows using "INSERT IGNORE INTO")
 #  - error: stop Lightning and report an error (i.e. insert rows using "INSERT INTO")
 #on-duplicate = "replace"
+# split regions into this size according to the importing data. This should be set to the same as tikv region size
+# to avoid tikv further split region. The default value is 96M.
```

Comment:
```suggestion
# Maximum KV size of SST files produced in the 'local' backend. This should be the same as
# the TiKV region size to avoid further region splitting. The default value is 96 MiB.
```